scripts/5_validator_tx.js

```diff
@@ -0,0 +1,63 @@
+require("dotenv").config();
```

Comment:
@gr8h 
can you rename the script and add comments to explain the purpose of the script

Reply:
renamed to `5_validator_operations.js`